package.json

```
@@ -2,7 +2,12 @@
   "name": "@react-native-community/geolocation",
   "version": "1.3.0",
   "description": "React Native Geolocation Module for iOS and Android",
-  "main": "js/index.js",
+  "react-native": "js/index.js",
+  "main": "web/commonjs/index.js",
+  "files": [
+    "web/",
```

Same comment as above.

```suggestion
    "lib/",
```